core/store/src/migrations.rs

```diff
@@ -749,8 +743,7 @@ pub fn migrate_28_to_29(path: &Path) {
     set_store_version(&store, 29);
 }
 
-#[cfg(feature = "protocol_feature_block_header_v3")]
-pub fn migrate_18_to_new_validator_stake(store: &Store) {
+pub fn migrate_29_to_30(path: &Path) {
```

Comment:
comment what this migration does (the V2-> V3 headers)

Reply:
My sense is that we commenting such things at the place of call